tests/10apidoc/30room-create.pl

```diff
@@ -313,7 +316,21 @@ sub matrix_create_room
    )->then( sub {
       my ( $body ) = @_;
 
-      Future->done( $body->{room_id}, $body->{room_alias} );
+      if (defined $body->{room_alias}) {
+         Future->done( $body->{room_id}, $body->{room_alias});
```

Comment:
If this field is non-standard, I'm wondering if we should just cut it out of SyTest altogether? Slightly easier to maintain (less branching), less Synapse-specific code in SyTest, everybody's a winner.

Reply:
That sounds like the thing to do to me. 👍

Reply:
I've removed it on https://github.com/matrix-org/sytest/pull/1216/commits/490735b1e1b41b6df4040f2000ad8b5987a40e39